translations/en-us.yaml

```diff
@@ -2806,7 +2806,7 @@ machine:
       sshPasswordLengthNotValid: The length of SSH password must between eight and thirty.
       sshPasswordInvalidCharacter: SSH password contains invalid characters.
       sshPasswordFormatError: "SSH password must contain at least three out of four kinds of following characters: uppercase letter, lowercase letters, numbers, and special characters."
-
+      nameNotValidForRancher: The name is invalid according to the Rancher hostname rule.
```

Comment:
Use `{settings.appName}` instead of hardcoding Rancher here. (add `settings: Ember.inject.service()` to the component if needed)